lgc/patch/PatchInOutImportExport.cpp

```diff
@@ -299,6 +354,29 @@ void PatchInOutImportExport::processShader() {
   }
 }
 
+// =====================================================================================================================
+// Visits all "call" instructions against the callee functions in current entry-point function.
+
```

Comment:
-> //
A blank line still has "//"

Reply:
> Just some minor coding comments.

Thanks for that, I think I fixed all of them.